plrust/src/plrust.rs

```diff
@@ -136,7 +136,15 @@ pub(crate) fn compile_function(fn_oid: pg_sys::Oid) -> eyre::Result<Output> {
     Ok(this_output.unwrap())
 }
 
-pub(crate) fn crate_name(db_oid: pg_sys::Oid, fn_oid: pg_sys::Oid) -> String {
+pub(crate) fn symbol_name(db_oid: pg_sys::Oid, fn_oid: pg_sys::Oid) -> String {
+    format!("plrust_fn_oid_{}_{}", db_oid.as_u32(), fn_oid.as_u32())
+}
+
+pub(crate) fn crate_name(
```

Comment:
Maybe these should explain when to use the different names?

Reply:
sure, I'll take care of that.